tests/unit/matrix/test_assembler.py

```diff
@@ -881,16 +906,16 @@ def _assemble_matrix_(self, request):
         return method_mock(request, Assembler, "_assemble_matrix")
 
     @pytest.fixture
-    def base_margin_(self, request):
-        return instance_mock(request, _UnweightedBaseMargin)
+    def margin_unweighted_base_(self, request):
+        return instance_mock(request, _MarginUnweightedBase)
 
     @pytest.fixture
     def _BaseOrderHelper_(self, request):
         return class_mock(request, "cr.cube.matrix.assembler._BaseOrderHelper")
 
     @pytest.fixture
-    def margin_proportion_(self, request):
-        return instance_mock(request, _MarginProportion)
+    def margin_table_proportion_(self, request):
```

Comment:
This one too. I realize it was wrongly positioned before you had occasion to touch it.